tests/nextdown.input

```diff
@@ -0,0 +1,31 @@
+# name nextdown
+# arg1 decimal
+# ret  decimal
+
+0                                                   -0.000001 decimal32
```

Comment:
Should this be returning -subnormal_min for all formats? e.g 0.000001e-DEC_MIN_EXP.

nextAfter may not be handling subnormals correctly.

Reply:
When the argument is 0 I think it makes sense the result being the smallest number possible.

But for normal numbers you can't increment subnormal values because you will need to change the number representation to have the smallest exponent possible and that will overflow the mantissa.

If you see in ISO-IEC TR 24732 2008-05-16 they have this `EPSILON` concept and I think they are using it for every number, zero included.
>the difference between 1 and the least value greater than 1 that is representable in the given floating point type:
DEC32_EPSILON  1E-6DF 
DEC64_EPSILON  1E-15DD 
DEC128_EPSILON 1E-33DL

If that is not the case and `nextAfter` is really broken I can create another PR and fix `nextAfter` before merging this PR.

Reply:
I believe format in this context implies decimal{32,64,128}. This function may or may not change the quantum exponent.  This is an implementation bug. subnormals make for tricky edge cases.